svc-aggregation/agmodel/system.go

```diff
@@ -948,3 +948,17 @@ func DeleteActiveRequest(key string) *errors.Error {
 	}
 	return nil
 }
+
+//SavePluginManagerInfo will save plugin manager  data into the database
+func SavePluginManagerInfo(body []byte, table string, key string) error {
+
+	conn, err := common.GetDBConnection(common.InMemory)
+	if err != nil {
+		return fmt.Errorf("error while trying to connecting to DB: %v", err.Error())
```

Comment:
Error message should convey that we are unable to save pluginmanager info

Reply:
This error will throw errors related to DB connection.
In next line, we use conn.create function, where error message will convey that we are unable to save pluginmanager info.

Reply:
err.Error() will add that. replace what you are adding here.